docs/integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/troubleshooting.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Troubleshooting
+---
+
+## Troubleshooting
+
+This section contains information to help you troubleshoot issues when installing Rancher Prime PAYG.
+
+Jobs and Pods:
+
+Check that pods or jobs have status Running/Completed
+
+
+```
+kubectl get pods --all-namespaces
+```
+
+
+If a pod is not in Running state, you can debug into the root cause by running:
```

Comment:
```suggestion
If a pod is not in a Running state, you can attempt to find the root cause with the following commands:
```

Suggest formatting the next few lines as bulleted list

Example --

- Describe pods: `kubectl describe pod <pod-name> -n <namespace>`